pkg/ast/sql/astsql.go

```diff
@@ -375,6 +369,9 @@ func parseSelect(astNode *structs.ASTNode, aggNode *structs.QueryAggregators, cu
 							measureOp = &structs.MeasureAggregator{MeasureCol: measureCol, MeasureFunc: getAggregationSQL(measureFunc, qid)}
 						}
 					} else {
+
+						leftExpr.Value = "0(" + leftExpr.Value + ")"
```

Comment:
What's happening here? Is the 0 supposed to be there?

Reply:
@AndrewHess Yes, it is zero because the aggregation functions like count, sum, etc., all have an id's from 1 to 6, but since the round is not an aggregation, the default will be zero, so when the actual rounding was done, the column is named as 0(column_Name), so for that reason I have changed the field Name for rounding function to match that. If you run the query with just `round(latitude),` then you will see there is a column that comes with `0(latitude)`.